--- teTeX-1.0/texk/share/texmf/fonts/source/public/misc/black.mf.cx	Tue Feb  6 23:56:09 2001
+++ teTeX-1.0/texk/share/texmf/fonts/source/public/misc/black.mf	Tue Feb  6 23:57:39 2001
@@ -1,2 +1,2 @@
-input blackcx % the `standard' black font is for the CX
+input blacklj % the `standard' black font is for the ljfour
 
--- teTeX-1.0/texk/share/texmf/fonts/source/public/misc/gray.mf.cx	Tue Feb  6 23:57:13 2001
+++ teTeX-1.0/texk/share/texmf/fonts/source/public/misc/gray.mf	Tue Feb  6 23:57:54 2001
@@ -1,2 +1,2 @@
-input graycx % the `standard' gray font is for the CX
+input graylj % the `standard' gray font is for the ljfour
 
