--- teTeX-1.0/texk/kpathsea/Makefile.in.orig	Wed Jun 16 07:44:37 1999
+++ teTeX-1.0/texk/kpathsea/Makefile.in	Thu Mar 15 17:43:55 2001
@@ -121,7 +121,7 @@
 stamp-paths: texmf.cnf
 	echo "/* paths.h: Generated from texmf.cnf. */" >paths.tmp
 	sed -e 's/%.*//' -e 's/^[ 	]*//' -e 's/[ 	]*$$//' texmf.cnf \
-	| grep '^[ 	]*[A-Z0-9_]*[ 	=]' \
+	| grep '^[ 	]*[[:upper:][:digit:]_]*[ 	=]' \
 	| sed '/^$$/d' \
 	| sed 's/^\([^ 	=]*\)[ 	]*=*[ 	]*\(.*\)/#ifndef DEFAULT_\1@#define DEFAULT_\1 "\2"@#endif/' \
 	| tr '@' '\012' \
