--- teTeX-src-beta-20020208/texk/web2c/tex.ch.wiget	Tue Mar  5 14:33:43 2002
+++ teTeX-src-beta-20020208/texk/web2c/tex.ch	Tue Mar  5 14:50:00 2002
@@ -192,8 +192,8 @@
 @d ssup_error_line = 255
 @d ssup_max_strings ==65535
 {Larger values may be used, but then the arrays consume much more memory.}
-@d ssup_trie_opcode == 65535
-@d ssup_trie_size == 262143
+@d ssup_trie_opcode == 262143
+@d ssup_trie_size == 1048575
 
 @d ssup_hyph_size == 65535 {Changing this requires changing (un)dumping!}
 @d iinf_hyphen_size == 610 {Must be not less than |hyph_prime|!}
--- teTeX-src-beta-20020208/texmf/web2c/texmf.cnf.wiget	Tue Mar  5 14:50:47 2002
+++ teTeX-src-beta-20020208/texmf/web2c/texmf.cnf	Tue Mar  5 14:52:44 2002
@@ -433,12 +433,13 @@
 % and German (for example).  To increase, you must change
 % `ssup_trie_opcode' and `ssup_trie_size' in tex.ch (and rebuild TeX);
 % the trie will then consume four bytes per entry, instead of two.
+% PLD note: now max is 262143
 % 
 % US English, German, and Portuguese: 30000.
 % German: 14000.
 % US English: 10000.
 % 
-trie_size = 64000
+trie_size = 131071
 
 % Buffer size.  TeX uses the buffer to contain input lines, but macro
 % expansion works by writing material into the buffer and reparsing the

--- teTeX-src-beta-20020530/texk/web2c/omegadir/com16bit.ch.wiget	Mon Jul  1 22:05:09 2002
+++ teTeX-src-beta-20020530/texk/web2c/omegadir/com16bit.ch	Mon Jul  1 22:07:04 2002
@@ -133,8 +133,8 @@
 @d ssup_error_line = 255
 @d ssup_max_strings ==200000
 {Larger values may be used, but then the arrays consume much more memory.}
-@d ssup_trie_opcode == 65535
-@d ssup_trie_size == 100000
+@d ssup_trie_opcode == 262143
+@d ssup_trie_size == 1048575
 
 @d ssup_hyph_size == 65535 {Changing this requires changing (un)dumping!}
 @d iinf_hyphen_size == 610 {Must be not less than |hyph_prime|!}

