--- teTeX-src-beta-20001218/texk/dvipsk/afm2tfm.c~	Mon Jan 10 22:11:50 2000
+++ teTeX-src-beta-20001218/texk/dvipsk/afm2tfm.c	Fri Dec 14 06:28:41 2001
@@ -1977,11 +1977,10 @@
  *   This routine prints out the line that needs to be added to psfonts.map.
  */
 void conspsfonts() {
-   (void)printf("%s %s", outname,
 #ifndef VMCMS
-   fontname) ;
+   (void)printf("%s %s", outname, fontname) ;
 #else /* VM/CMS: fontname is ascii, so we use ebfontname */
-   ebfontname) ;
+   (void)printf("%s %s", outname, ebfontname) ;
 #endif
    if (slantparam || efactorparam || inenname) {
       (void)printf(" \"") ;
