--- teTeX-src-beta-20020922/texk/web2c/Makefile.in.wiget	Thu May 30 18:31:16 2002
+++ teTeX-src-beta-20020922/texk/web2c/Makefile.in	Thu Sep 26 20:45:03 2002
@@ -417,7 +417,7 @@
 all_bases = #mf.base $(bases)
 all_mems = #mpost.mem $(mems)
 
-dumpenv = TEXMFCNF=../kpathsea TEXMF=$(texmf)
+dumpenv = TEXMFCNF=../kpathsea TEXMF=../share/texmf
 
 dumps: fmts efmts cfmts ofmts pdffmts pdfefmts bases mems
 fmts: $(all_fmts)
--- teTeX-src-beta-20020922/Makefile.in.wiget	Sun Sep 15 14:47:50 2002
+++ teTeX-src-beta-20020922/Makefile.in	Thu Sep 26 20:46:29 2002
@@ -83,6 +83,8 @@
 	     (cd $$dir && PATH=$(scriptdir):$(bindir):$(PATH) \
                 $(MAKE) $@) || exit 1; \
 	done
+
+init:
 	$(srcdir)/selfautofix "$(bindir)" "$(texmf)/web2c"
 	cd $(bindir); for i in $(ALLPROGS); do test -f $$i && { echo strip $$i; strip $$i || true; }; done; true
 	test -f $(scriptdir)/mktexlsr && \
