--- tetex-src-2.0.2/texk/xdvik/configure~	Tue Feb 18 05:12:53 2003
+++ tetex-src-2.0.2/texk/xdvik/configure	Sun Mar 30 18:04:56 2003
@@ -4396,11 +4396,11 @@
 
 
 if test "x$with_x_toolkit" = xxaw3d; then
-   x_tool_libs="-lXaw3d -lXmu -lXt"
+   x_tool_libs="-lXaw3d -lXmu -lXt -lXpm"
 elif test "x$with_x_toolkit" = xneXtaw; then
-   x_tool_libs="-lneXtaw -lXmu -lXt"
+   x_tool_libs="-lneXtaw -lXmu -lXt -lXpm"
 else
-  x_tool_libs="-lXaw -lXmu -lXt"
+  x_tool_libs="-lXaw -lXmu -lXt -lXpm"
 fi
 
 
--- tetex-src-2.0.2/texk/xdvik/configure.in~	Sun Mar 30 18:04:18 2003
+++ tetex-src-2.0.2/texk/xdvik/configure.in	Sun Mar 30 18:05:07 2003
@@ -74,11 +74,11 @@
 AC_SUBST(x_ext_lib)
 
 if test "x$with_x_toolkit" = xxaw3d; then
-   x_tool_libs="-lXaw3d -lXmu -lXt"
+   x_tool_libs="-lXaw3d -lXmu -lXt -lXpm"
 elif test "x$with_x_toolkit" = xneXtaw; then
-   x_tool_libs="-lneXtaw -lXmu -lXt"
+   x_tool_libs="-lneXtaw -lXmu -lXt -lXpm"
 else
-  x_tool_libs="-lXaw -lXmu -lXt"
+  x_tool_libs="-lXaw -lXmu -lXt -lXpm"
 fi
 AC_SUBST(x_tool_libs)
 
