--- teTeX-1.0.new/texk/klibtool	Mon Jul 19 12:46:27 1999
+++ teTeX-1.0/texk/klibtool	Thu Jan 14 19:56:54 1999
@@ -268,7 +268,7 @@
     case $host_type in
       *-*-linux*)
         shared_support=true
-        SHARED_postinstall='ldconfig $libdir'
+        SHARED_postinstall='/sbin/ldconfig $libdir || :'
         ;;
         
       *-*-solaris2*)
@@ -290,7 +290,7 @@
       *-*-sunos4*)
         shared_support=true
         STATIC_ranlib=ranlib
-        SHARED_postinstall='ldconfig $libdir'
+        SHARED_postinstall='/sbin/ldconfig $libdir || |:'
         if test $compiler = cc; then
           args_SHARED_compile=-PIC
           prog_SHARED_archive=ld
@@ -310,7 +310,7 @@
 
       *-*-freebsd*)
         shared_support=true
-        SHARED_postinstall='ldconfig -m $libdir'
+        SHARED_postinstall='/sbin/ldconfig -m $libdir || :'
         ;;
 
       hppa*-*-hpux*)
