--- tetex-src-beta-2.99.1.20041026/texk/web2c/fmtutil.in.wiget	2004-10-05 07:08:28.000000000 +0200
+++ tetex-src-beta-2.99.1.20041026/texk/web2c/fmtutil.in	2004-11-03 11:23:08.941701413 +0100
@@ -38,11 +38,11 @@
 
 # Change "tex.ini -> bplain.ini" and "- -> language.dat"
 # if you want babel support in tex:
-tex		tex		-		-translate-file=cp227.tcx tex.ini
+tex		tex		language.dat		-translate-file=cp227.tcx bplain.ini
 @ALEPH@aleph		aleph		-		*aleph.ini
 @PETEX@latex		pdfetex		language.dat	-translate-file=cp227.tcx *latex.ini
 @PETEX@etex		pdfetex		language.def	-translate-file=cp227.tcx *etex.ini
-@PETEX@pdftex		pdfetex		-	-translate-file=cp227.tcx *pdftex.ini
+@PETEX@pdftex		pdfetex		language.dat	-translate-file=cp227.tcx *pdfbplain.ini
 @PETEX@pdflatex	pdfetex		language.dat	-translate-file=cp227.tcx *pdflatex.ini
 @PETEX@pdfetex		pdfetex		language.def	-translate-file=cp227.tcx *pdfetex.ini
 @OMEGA@omega		omega		-	omega.ini
@@ -53,32 +53,32 @@
 
 # Change "amstex.ini -> bamstex.ini" and "- -> language.dat"
 # if you want babel support in amstex:
-#! amstex	pdfetex		-		-translate-file=cp227.tcx *amstex.ini
+amstex	pdfetex		language.dat		-translate-file=cp227.tcx *bamstex.ini
 
 # Change "pdfamstex.ini -> pdfbamstex.ini" and "- -> language.dat"
 # if you want babel support in pdfamstex:
-#! pdfamstex	pdfetex			-	-translate-file=cp227.tcx *pdfamstex.ini
+pdfamstex	pdfetex			language.dat	-translate-file=cp227.tcx *pdfbamstex.ini
 
 # Context formats:
 #! cont-cz	pdfetex		cont-usr.tex	-translate-file=cp227.tcx *cont-cz.ini
-#! cont-de	pdfetex		cont-usr.tex	-translate-file=cp227.tcx *cont-de.ini
+cont-de	pdfetex		cont-usr.tex	-translate-file=cp227.tcx *cont-de.ini
 @PETEX@cont-en	pdfetex		cont-usr.tex	-translate-file=cp227.tcx *cont-en.ini
-#! cont-nl	pdfetex		cont-usr.tex	-translate-file=cp227.tcx *cont-nl.ini
-#! cont-uk	pdfetex		cont-usr.tex	-translate-file=cp227.tcx *cont-uk.ini
+cont-nl	pdfetex		cont-usr.tex	-translate-file=cp227.tcx *cont-nl.ini
+cont-uk	pdfetex		cont-usr.tex	-translate-file=cp227.tcx *cont-uk.ini
 
 # language-specific things:
 
 # - Cyrillic:
-#! cyrtex	pdfetex		language.dat	-translate-file=cp227.tcx *cyrtex.ini
-#! cyrtexinfo	pdfetex		language.dat	-translate-file=cp227.tcx *cyrtxinf.ini
-#! cyramstex	pdfetex		language.dat	-translate-file=cp227.tcx *cyramstx.ini
+cyrtex	pdfetex		language.dat	-translate-file=cp227.tcx *cyrtex.ini
+cyrtexinfo	pdfetex		language.dat	-translate-file=cp227.tcx *cyrtxinf.ini
+cyramstex	pdfetex		language.dat	-translate-file=cp227.tcx *cyramstx.ini
 
 # - Polish:
-#! platex		pdfetex		language.dat	-translate-file=cp227.tcx *platex.ini
-#! pdfplatex		pdfetex		language.dat	-translate-file=cp227.tcx *pdfplatex.ini
-#! mex		pdfetex		mexconf.tex	-translate-file=cp227.tcx *mex.ini
-#! pdfmex		pdfetex		mexconf.tex	-translate-file=cp227.tcx *pdfmex.ini
-#! utf8mex		pdfetex		mexconf.tex	-enc *utf8mex.ini
+platex		pdfetex		language.dat	-translate-file=cp227.tcx *platex.ini
+pdfplatex		pdfetex		language.dat	-translate-file=cp227.tcx *pdfplatex.ini
+mex		pdfetex		mexconf.tex	-translate-file=cp227.tcx *mex.ini
+pdfmex		pdfetex		mexconf.tex	-translate-file=cp227.tcx *pdfmex.ini
+utf8mex		pdfetex		mexconf.tex	-enc *utf8mex.ini
 
 # - Polish with translated messages (experimental):
 #! platex-pl	pdfetex		-		nls=tex-pl,il2-pl *platex.ini
@@ -86,16 +86,16 @@
 #! pdfmex-pl	pdfetex		mexconf.tex	nls=pdftex-pl,il2-pl *pdfmex.ini
 
 # - Czech / Slovak:
-#! csplain	pdfetex		-		-translate-file=cp227.tcx *csplain.ini
-#! cslatex	pdfetex		-	-translate-file=cp227.tcx *cslatex.ini
-#! pdfcsplain	pdfetex		-		-translate-file=cp227.tcx *csplain.ini
-#! pdfcslatex	pdfetex		-    -translate-file=cp227.tcx *cslatex.ini
+csplain	pdfetex		-		-translate-file=cp227.tcx *csplain.ini
+cslatex	pdfetex		-	-translate-file=cp227.tcx *cslatex.ini
+pdfcsplain	pdfetex		-		-translate-file=cp227.tcx *csplain.ini
+pdfcslatex	pdfetex		-    -translate-file=cp227.tcx *cslatex.ini
 
 # Other formats:
 
 # Change "eplain.ini -> ebplain.ini" and "- -> language.dat"
 # if you want babel support in eplain:
-#! eplain	pdfetex		-	-translate-file=cp227.tcx *eplain.ini
+eplain	pdfetex		language.dat	-translate-file=cp227.tcx *ebplain.ini
 
 
 # mptopdf utility
