--- tetex-src-2.0/texk/web2c/tex.ch	Fri Jan 24 21:55:58 2003
+++ tetex-src-2.0/texk/web2c/tex.ch.new	Tue Feb  4 22:28:47 2003
@@ -199,8 +199,8 @@
 @d ssup_error_line = 255
 @d ssup_max_strings == 262143
 {Larger values than 65536 cause the arrays consume much more memory.}
-@d ssup_trie_opcode == 65535
-@d ssup_trie_size == 262143
+@d ssup_trie_opcode == 262143
+@d ssup_trie_size == 1048575
 
 @d ssup_hyph_size == 65535 {Changing this requires changing (un)dumping!}
 @d iinf_hyphen_size == 610 {Must be not less than |hyph_prime|!}
