--- tetex-src-beta-2.96.7.20040721/texk/xdvik/Makefile.in.orig	2004-07-15 22:34:42.000000000 +0200
+++ tetex-src-beta-2.96.7.20040721/texk/xdvik/Makefile.in	2004-09-11 16:18:27.914101232 +0200
@@ -102,7 +102,7 @@
 	cd gui; $(MAKE) $(makeargs) libgui.a
 
 @final_exec_name@: $(kpathsea) $(objects) $(LIBT1DEP) $(LIBGUIDEP)
-	$(kpathsea_link) $(objects) $(LIBGUI) $(x_link) $(LOADLIBES)
+	$(kpathsea_link) $(objects) $(LIBGUI) $(LOADLIBES) $(x_link)
 
 squeeze.o: squeeze.c
 	$(build_compile) -c $<
