--- teTeX-src-beta-20020208/texk/kpathsea/texmf.in.wiget	Thu Feb 21 01:53:17 2002
+++ teTeX-src-beta-20020208/texk/kpathsea/texmf.in	Thu Feb 21 02:02:50 2002
@@ -396,7 +396,7 @@
 % 
 main_memory.context = 1500000
 main_memory.mpost = 1000000
-main_memory = 263000 % words of inimemory available; also applies to inimf&mp
+main_memory = 2500000 % words of inimemory available; also applies to inimf&mp
 extra_mem_top = 0    % extra high memory for chars, tokens, etc.
 extra_mem_bot = 0    % extra low memory for boxes, glue, breakpoints, etc.
 
@@ -411,22 +411,22 @@
 % Extra space for the hash table of control sequences (which allows 10K
 % names as distributed).
 hash_extra.context = 40000
-hash_extra = 0
+hash_extra = 40000
 
 % Max number of characters in all strings, including all error messages,
 % help texts, font names, control sequences.  These values apply to TeX and MP.
 pool_size.context = 750000
-pool_size = 125000		
+pool_size = 750000		
 % Minimum pool space after TeX/MP's own strings; must be at least
 % 25000 less than pool_size, but doesn't need to be nearly that large.
 string_vacancies.context = 45000
-string_vacancies = 25000
+string_vacancies = 45000
 % Maximum number of strings.
 max_strings.context = 55000
-max_strings = 15000
+max_strings = 55000
 % min pool space left after loading .fmt
 pool_free.context = 47500
-pool_free = 5000
+pool_free = 47500
 
 % Hyphenation trie. As distributed, the maximum is 65535; this should
 % work unless `unsigned short' is not supported or is smaller than 16
@@ -439,7 +439,7 @@
 % German: 14000.
 % US English: 10000.
 % 
-trie_size = 64000
+trie_size = 200000
 
 % Buffer size.  TeX uses the buffer to contain input lines, but macro
 % expansion works by writing material into the buffer and reparsing the
@@ -450,14 +450,14 @@
 
 hyph_size = 1000        % number of hyphenation exceptions, >610 and <32767.
 nest_size.context = 500
-nest_size = 100		% simultaneous semantic levels (e.g., groups)
+nest_size = 500		% simultaneous semantic levels (e.g., groups)
 max_in_open = 15	% simultaneous input files and error insertions
 param_size.context = 1500
-param_size = 500	% simultaneous macro parameters
-save_size.context = 5000
-save_size = 4000	% for saving values outside current group
+param_size = 1500	% simultaneous macro parameters
+save_size.context = 10000
+save_size = 10000	% for saving values outside current group
 stack_size.context = 1500
-stack_size = 300	% simultaneous input sources
+stack_size = 1500	% simultaneous input sources
 
 % These are Omega-specific.
 ocp_buf_size = 20000	% character buffers for ocp filters.
